cfgov/unprocessed/css/atoms/multi-select.less

```diff
@@ -0,0 +1,124 @@
+.cf-multi-select {
+    .webfont-regular();
+
+    position: relative;
+
+    select {
+        display: none;
+    }
+
+    &_header {
+        position: relative;
+    }
+
+    &_search[type="text"] {
+        display: block;
+
+        box-sizing: border-box;
+        width: 100%;
+
+        .active & {
+            border-color: @pacific;
+            outline: 1px solid @pacific;
+        }
+    }
+
+    &_fieldset {
+        /* Resets */
+        border-color: @gray;
+        border-top: none;
+        margin: 0;
+        padding: 0;
+
+        /* Styles */
+        box-sizing: border-box;
+        overflow: scroll;
+        position: absolute;
+        z-index: 10;
+
+        max-height: 0;
+        margin-top: -1px;
+        width: 100%;
+
+        background-color: @white;
+        transition: max-height 0.25s ease-out;
+    }
+
+     &.active {
+        .cf-multi-select_fieldset {
+            margin-top: 0;
+            max-height: 140px;
+
+            border-color: @pacific;
+            border-top: 1px solid;
+            outline: 1px solid @pacific;
+        }
+    }
+
+    &_options {
+        padding: unit(10px / @base-font-size-px, em);
+
+        li:last-child {
+            margin-bottom: 0;
+        }
+
+        &.filtered li:not( .filter-match ) {
```

Comment:
Hm, it seems more intuitive to set the class on the matched one, but if this is the only IE8 incompatibility it may be worthwhile. I think we could just turn it off for IE8 too.